.github/workflows/contracts.yml

```diff
@@ -258,7 +258,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
         with:
-          module: "github.com/keep-network/keep-core/solidity"
+          module: "github.com/keep-network/keep-core/solidity-v1"
```

Comment:
This will require an update in https://github.com/keep-network/ci/blob/main/config/config.json as well.

Reply:
yea, good point https://github.com/keep-network/ci/pull/21